microcosm_pubsub/dispatcher.py

```diff
@@ -36,19 +39,28 @@ def __init__(self, graph):
         self.send_metrics = graph.pubsub_send_metrics
         self.send_batch_metrics = graph.pubsub_send_batch_metrics
         self.max_processing_attempts = graph.config.sqs_message_dispatcher.message_max_processing_attempts
+        self.loop = asyncio.get_event_loop()
 
     def handle_batch(self, bound_handlers) -> List[MessageHandlingResult]:
         """
         Send a batch of messages to a function.
 
         """
+        executor = concurrent.futures.ThreadPoolExecutor(5)
```

Comment:
we should add this to __init__ so we are not recreating this over and over. also the magic number should come out.